src/test/scala/org/camunda/feel/impl/interpreter/InterpreterNonExistingVariableExpressionTest.scala

```diff
@@ -16,82 +16,124 @@
-    eval(""" x <= "string" """) should be(ValNull)
-    eval(""" "string" <= x """) should be(ValNull)
-    eval("x <= null") should be(ValNull)
-    eval("null <= x") should be(ValNull)
-    eval("x <= y") should be(ValNull)
+    evaluateExpression("x <= 1") should returnNull()
+    evaluateExpression("1 <= x") should returnNull()
+    evaluateExpression("x <= true") should returnNull()
+    evaluateExpression("true <= x") should returnNull()
+    evaluateExpression(""" x <= "string" """) should returnNull()
+    evaluateExpression(""" "string" <= x """) should returnNull()
+    evaluateExpression("x <= null") should returnNull()
+    evaluateExpression("null <= x") should returnNull()
+    evaluateExpression("x <= y") should returnNull()
   }
 
   it should "compare with `>=`" in {
-    eval("x >= 1") should be(ValNull)
-    eval("1 >= x") should be(ValNull)
-    eval("x >= true") should be(ValNull)
-    eval("true >= x") should be(ValNull)
-    eval(""" x >= "string" """) should be(ValNull)
-    eval(""" "string" >= x """) should be(ValNull)
-    eval("x >= null") should be(ValNull)
-    eval("null >= x") should be(ValNull)
-    eval("x >= y") should be(ValNull)
+    evaluateExpression("x >= 1") should returnNull()
+    evaluateExpression("1 >= x") should returnNull()
+    evaluateExpression("x >= true") should returnNull()
+    evaluateExpression("true >= x") should returnNull()
+    evaluateExpression(""" x >= "string" """) should returnNull()
+    evaluateExpression(""" "string" >= x """) should returnNull()
+    evaluateExpression("x >= null") should returnNull()
+    evaluateExpression("null >= x") should returnNull()
+    evaluateExpression("x >= y") should returnNull()
   }
 
   it should "compare with `between _ and _`" in {
-    eval("x between 1 and 3") should be(ValNull)
-    eval("1 between x and 3") should be(ValNull)
-    eval("3 between 1 and x") should be(ValNull)
-    eval("x between y and 3") should be(ValNull)
-    eval("x between 1 and y") should be(ValNull)
-    eval("x between y and z") should be(ValNull)
+    evaluateExpression("x between 1 and 3") should returnNull()
+    evaluateExpression("1 between x and 3") should returnNull()
+    evaluateExpression("3 between 1 and x") should returnNull()
+    evaluateExpression("x between y and 3") should returnNull()
+    evaluateExpression("x between 1 and y") should returnNull()
+    evaluateExpression("x between y and z") should returnNull()
   }
+
+  it should "return null" in {
+    evaluateExpression("non_existing") should (
+      returnNull() and reportFailure(
+        failureType = EvaluationFailureType.NO_VARIABLE_FOUND,
+        failureMessage = "No variable found with name 'non_existing'")
+      )
+  }
+
+  "A non-existing input value" should "be equal to null" in {
+    // use parse + evaluate to avoid setting an input value
+    val parsedExpression = engine.parseUnaryTests("null").parsedExpression
+
+    engine.evaluate(parsedExpression) should (
```

Comment:
💭 What about creating a new `evaluateParsedExpression` helper method in `FeelEngineTest`. So we can align all the test class with the new test engine. Improving also the readability